.devcontainer/Dockerfile

```diff
@@ -1,20 +1,17 @@
 # For details, see https://github.com/devcontainers/images/tree/main/src/ruby
 FROM mcr.microsoft.com/devcontainers/ruby:1-3.3-bookworm
 
-# Install Rails
-# RUN gem install rails webdrivers
-
+# Update existing node version, keep in sync with .nvmrc
 ARG NODE_VERSION="20"
-RUN su vscode -c "source /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"
```

Comment:
We removed the "vscode" user elsewhere, because hypothetically people can use this as a non-devcontainers docker compose flow.